src/Mvc/src/Microsoft.AspNetCore.Mvc.Core/MvcCoreLoggerExtensions.cs

```diff
@@ -156,498 +156,499 @@ static MvcCoreLoggerExtensions()
         {
             _actionExecuting = LoggerMessage.Define<string, string>(
                 LogLevel.Information,
-                1,
+                new EventId(1, "ActionExecuting"),
                 "Route matched with {RouteData}. Executing action {ActionName}");
 
             _actionExecuted = LoggerMessage.Define<string, double>(
                 LogLevel.Information,
-                2,
+                new EventId(2, "ActionExecuted"),
                 "Executed action {ActionName} in {ElapsedMilliseconds}ms");
 
             _pageExecuting = LoggerMessage.Define<string, string>(
                 LogLevel.Information,
-                3,
+                new EventId(3, "PageExecuting"),
                 "Route matched with {RouteData}. Executing page {PageName}");
 
             _pageExecuted = LoggerMessage.Define<string, double>(
                 LogLevel.Information,
-                4,
+                new EventId(4, "PageExecuted"),
                 "Executed page {PageName} in {ElapsedMilliseconds}ms");
 
             _challengeResultExecuting = LoggerMessage.Define<string[]>(
                 LogLevel.Information,
-                1,
+                new EventId(1, "ChallengeResultExecuting"),
                 "Executing ChallengeResult with authentication schemes ({Schemes}).");
 
             _contentResultExecuting = LoggerMessage.Define<string>(
                 LogLevel.Information,
-                1,
+                new EventId(1, "ContentResultExecuting"),
                 "Executing ContentResult with HTTP Response ContentType of {ContentType}");
 
             _actionMethodExecuting = LoggerMessage.Define<string, ModelValidationState>(
                 LogLevel.Information,
-                1,
+                new EventId(1, "ActionMethodExecuting"),
                 "Executing action method {ActionName} - Validation state: {ValidationState}");
 
             _actionMethodExecutingWithArguments = LoggerMessage.Define<string, string[], ModelValidationState>(
                 LogLevel.Information,
-                1,
+                new EventId(1, "ActionMethodExecutingWithArguments"),
                 "Executing action method {ActionName} with arguments ({Arguments}) - Validation state: {ValidationState}");
 
             _actionMethodExecuted = LoggerMessage.Define<string, string, double>(
                 LogLevel.Information,
-                2,
+                new EventId(2, "ActionMethodExecuted"),
                 "Executed action method {ActionName}, returned result {ActionResult} in {ElapsedMilliseconds}ms.");
 
             _logFilterExecutionPlan = LoggerMessage.Define<string, string[]>(
                 LogLevel.Debug,
-                1,
+                new EventId(1, "LogFilterExecutionPlan"),
```

Comment:
This name doesn't look like any of the others. Suggest `FilterExecutionPlan` or `OrderedFilterExecutionPlan`.